src/NServiceBus.AcceptanceTests/Performance/Receive/Receive_performance.cs

```diff
@@ -32,6 +33,7 @@ public void With_dtc_suppressed()
                     .WithEndpoint<ReceiveEndpoint>(b =>
                         {
                             b.CustomConfig(c => Configure.Transactions.Advanced(a => a.DisableDistributedTransactions()));
+                            b.CustomConfig(configure => configure.SuppressOutdatedTimeoutDispatchWarning());
```

Comment:
why calling CustomConfig twice instead of converting the first lambda expression to a statement body? (e.g. c => { ... })